nano/templates/communication.tmpl

```diff
@@ -21,6 +21,8 @@
 		{{empty}}
 			<div class=item>There are no messages.</div>
 		{{/for}}
+		<div class=item>{{if data.have_printer}}{{:helper.link( !data.message_printing_intercepts ? 'Turn on intercept printing' : 'Turn off intercept printing', null, {'action' : 'toggleintercept'})}}{{/if}}</div>
```

Comment:
"Turn on intercept printing" sounds quite technical for a application used by command staff.
I´d go for something like "Enable/Disable report printing"